seqr/management/commands/check_for_new_samples_from_pipeline.py

```diff
@@ -242,6 +249,10 @@ def _load_new_samples(cls, metadata_path, genome_version, dataset_type, run_vers
         except Exception as e:
             logger.error(f'Error reporting loading failure for {run_version}: {e}')
 
+        # Update sample qc
+        if 'sample_qc' in metadata:
+            cls.update_individuals_sample_qc(sample_type, updated_samples, metadata['sample_qc'])
```

Comment:
we should wrap this in a try/except block that logs an error because if something goes wrong somehow we would still want the rest of the job to proceed